domains/nayem.domain.json

```diff
@@ -0,0 +1,18 @@
+{
+	"$schema": "../schemas/domain.schema.json",
+
+	"description": "As an Website and Android Application Developer I need a Website. For this purpose I need this domain.",
+
+	"domain": "is-cool.dev",
+	"subdomain": "nayem",
+
+	"owner": {
+		"email": "nayemahmedniloy@gmail.com"
+	},
+
+	"record": {
+		"NS": ["ns1.epizy.com", "ns2.epizy.com"]
```

Comment:
Please provide the specific reason you need NS records.

Reply:
I am sorry. I have changed the file name now. Will it be okay?
...........................................................................................................................
I am using **infinityfree** hosting service. To use a custom domain the hosting service requires to use these Name Server (NS) records. That's why I need these NS records with the domain.